huddle/src/main/java/team/uptech/huddle/builder/FontBuilder.kt

```diff
@@ -0,0 +1,18 @@
+package team.uptech.huddle.builder
+
+import team.uptech.huddle.util.Constants
+import team.uptech.huddle.util.ElementMarker
+
+/**
+ * Base DSL Builder for TODO: add docs
+ *
+ * @property titleFont the font resource for the dialog title
+ * @property messageFont the font resource for the dialog message
+ * @property ctaFont the font resource for the dialog action button
+ */
+@ElementMarker
+class FontBuilder {
+  var titleFont: Int = Constants.DEFAULT_FONT
+  var messageFont: Int = Constants.DEFAULT_FONT
+  var ctaFont: Int = Constants.DEFAULT_FONT
```

Comment:
Please import as static `Constants. DEFAULT_FONT`
```suggestion
  var titleFont: Int = DEFAULT_FONT
  var messageFont: Int = DEFAULT_FONT
  var ctaFont: Int = DEFAULT_FONT
```

Reply:
Do i need to rename titleFont to titleRes?

Reply:
Nope, it's ok to keep like that

Reply:
but you can reduce it to just `title`

Reply:
Use case:
```kotlin
font {
  title = R.font.*
}
```

Reply:
👍